cli/src/config/templates.toml

```diff
@@ -39,6 +39,20 @@ log = 'builtin_log_compact'
 op_log = 'builtin_op_log_compact'
 show = 'builtin_log_detailed'
 
+op_log_node = 'if(current_operation, "@", "◉")'
+
+log_node = '''
+label("node",
+  coalesce(
+    if(!self, label("elided", "⇋")),
+    if(current_working_copy, label("wcc", "@")),
+    if(immutable, label("immutable", "◆")),
+    if(description.starts_with("wip: "), label("wip", "!")),
```

Comment:
while it's nice to have "wip:" in documented example, I don't think it should be included in the default template. We don't have a canonical form of WIP labeling.